conda/cli/install.py

```diff
@@ -223,10 +223,19 @@ def install(args, parser, command='install'):
 
             try:
                 assert len(vers_inst) == 1, name
+            except AssertionError as e:
```

Comment:
you don't need the whole `try/assert/except` thing here.  Just

```
if len(vers_inst) != 1:
    msg = "Number of versions for %s" % name
    raise CondaAssertionError(msg, 1, len(vers_inst)) 
```

or something like that.  We can do a lot better than that still probably.